test/jdk/java/awt/Checkbox/CheckboxIndicatorSizeTest.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 1999, 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+import java.awt.Checkbox;
+import java.awt.CheckboxGroup;
+import java.awt.Color;
+import java.awt.Font;
+import java.awt.Frame;
+import java.awt.GridLayout;
+import java.awt.Label;
+import java.awt.Menu;
+import java.awt.MenuBar;
+import java.awt.MenuItem;
+import java.awt.Panel;
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+/*
+ * @test
+ * @bug 4090493
+ * @summary Test for Checkbox indicator size
+ * @library /java/awt/regtesthelpers
+ * @build PassFailJFrame
+ * @run main/manual CheckboxIndicatorSizeTest
+ */
+
+public class CheckboxIndicatorSizeTest implements ActionListener {
+    private static final String INSTRUCTIONS = """
+            Indicator size of Checkbox depends
+            on the platform font used to render the label.
+
+            In the frame you can see a group of checkboxes
+            and radio buttons.
+            Verify that all checkboxes has indicator of the same
+            size and all radio buttons has indicator of the same size.
+            You can use menu to change the font size.
+
+            If the bug is present, the checkbox/radiobutton with dingbats label
+            will have a smaller indicator.
+
+            TEST DIRECTIONS.
+            You can use 'FontSize' menu to change font size.
+            Verify that all indicators of checkboxes has the same size.
+            Verify that all indicators of radio buttons has the same size.
+            Press PASS if the above conditions are true else Press FAIL.
```

Comment:
I guess "Same size" is not correct here...I guess what we meant is the size should be proportionate to scaling or "size will change if font size is changed", it seems the size does not increase if we increase the scale..
but it still remains the "same", so some tester can consider it pass

Reply:
I have a clarification about this: 
Does thecheckbox size depend on label size (font size) and they need to be scaled proportionately or is the size dependent only on uiScale?

Reply:
One is global setting and the other local..and local would get affected by global setting ie
Checkboz size should be proportionate to local label fontsize and 
when we up the global uiScale, it would also increase the label font implicitly which then would affect checkbox size